website/static/js/logFeed.js

```diff
@@ -176,6 +176,7 @@ var createLogs = function(logData){
             nodeCategory: item.node.category,
             contributors: item.contributors,
             nodeUrl: item.node.url,
+            projectUrl: item.node.node_type === 'project' ? '/' + item.node.registered_from_id + '/' : item.node.url,
```

Comment:
What if `item.node.node_type === 'project' ` but it is not a registration? Should this check for `item.node.registered_from_id` instead?